src/state/ClaimedRewardToastState.ts

```diff
@@ -0,0 +1,24 @@
+import { Reward } from '@hyperplay/utils'
+import { makeAutoObservable } from 'mobx'
+
+class ClaimedRewardToastState {
+  private reward: Reward | null = null
+
+  constructor() {
+    makeAutoObservable(this)
+  }
+
+  showClaimedReward(reward: Reward) {
+    this.reward = reward
+  }
```

Comment:
can we add a 5 second timeout here to clear the reward? and if it is claimed while this timer is active, it resets back to 5 more seconds to timeout